tests/unit/test_schemas_json_load.py

```diff
@@ -8,270 +8,763 @@
+    assert data == valid_full
+
+    invalid_no_name = {
+        "identifier": "entity-one",
+        "scheme": "entity-id-scheme"
+    }
+    with pytest.raises(ValidationError):
+        data = AffiliationSchemaV1().load(invalid_no_name)
+
+    invalid_no_identifier = {
+        "name": "Entity One",
+        "scheme": "entity-id-scheme"
+    }
+    with pytest.raises(ValidationError):
+        data = AffiliationSchemaV1().load(invalid_no_identifier)
+
+    invalid_no_scheme = {
+        "name": "Entity One",
+        "identifier": "entity-one"
+    }
+    with pytest.raises(ValidationError):
+        data = AffiliationSchemaV1().load(invalid_no_scheme)
+
+
+def test_creator():
+    """Test creator schema."""
+    valid_minimal = {
+        "name": "Julio Cesar",
+        "type": "Personal",
+        "given_name": "Julio",
+        "family_name": "Cesar",
+        "identifiers": [{
+            "identifier": "9999-9999-9999-9999",
+            "scheme": "Orcid"
+        }],
+        "affiliations": [{
+            "name": "Entity One",
+            "identifier": "entity-one",
+            "scheme": "entity-id-scheme"
+        }]
+    }
+
+    data = CreatorSchemaV1().load(valid_minimal)
+    assert data == valid_minimal
+
+    valid_minimal = {
+        "name": "Julio Cesar",
+        "type": "Personal"
+    }
+
+    data = CreatorSchemaV1().load(valid_minimal)
+    assert data == valid_minimal
+
+    invalid_no_name = {
+        "type": "Personal",
+        "given_name": "Julio",
+        "family_name": "Cesar",
+        "identifiers": [{
+            "identifier": "9999-9999-9999-9999",
+            "scheme": "Orcid"
+        }],
+        "affiliations": [{
+            "name": "Entity One",
+            "identifier": "entity-one",
+            "scheme": "entity-id-scheme"
+        }]
+    }
+    with pytest.raises(ValidationError):
+        data = CreatorSchemaV1().load(invalid_no_name)
+
+    invalid_no_name_type = {
+        "name": "Julio Cesar",
+        "given_name": "Julio",
+        "family_name": "Cesar",
+        "identifiers": [{
+            "identifier": "9999-9999-9999-9999",
+            "scheme": "Orcid"
+        }],
+        "affiliations": [{
+            "name": "Entity One",
+            "identifier": "entity-one",
+            "scheme": "entity-id-scheme"
+        }]
+    }
+    with pytest.raises(ValidationError):
+        data = CreatorSchemaV1().load(invalid_no_name_type)
+
+    invalid_name_type = {
+        "name": "Julio Cesar",
+        "type": "Invalid",
+        "given_name": "Julio",
+        "family_name": "Cesar",
+        "identifiers": [{
+            "identifier": "9999-9999-9999-9999",
+            "scheme": "Orcid"
+        }],
+        "affiliations": [{
+            "name": "Entity One",
+            "identifier": "entity-one",
+            "scheme": "entity-id-scheme"
+        }]
+    }
+    with pytest.raises(ValidationError):
+        data = CreatorSchemaV1().load(invalid_name_type)
```

Comment:
A test for `"type": "Organizational"` would be good: what happens to affiliations and `given/family_name`s in that case?

Reply:
They wouldn't be there. We need to increase validation of this kind of stuff later on the road.

Reply:
Thanks for creating https://github.com/inveniosoftware/invenio-rdm-records/issues/52 !